### ISSUE TITLE
Can't detect my USB drive

### ISSUE THREAD

**Issue by Justinzobel** [thefanclub/dd-utility#1]
grabbed the ddutility-1.4.sh from here, chmod +x, insert USB drive, ran it, selected ISO, tells me it can't find my removable drive but everything else sees it.

Output of lsblk:
sde      8:64   1   7.5G  0 disk 
├─sde1   8:65   1   712M  0 part 
└─sde2   8:66   1  17.2M  0 part

**Comment by thefanclub** [comment]
```
Hi Justin,

Thank you for the feedback.

What linux flavour are you using ?

The utility looks for mounted devices on /media/username

On some flavours you can change line 38 :

from :

mountpath="/media/$SUDO_USER"

to :

mountpath="/media"

Please send us the output of : 

df

with the card mounted.

Have a great day
Christiaan





    CHRISTIAAN DIEDERICKS | www.thefanclub.co.za |
    info@thefanclub.co.za 



On 2015/05/27 6:03 AM, Justin wrote:


  grabbed the ddutility-1.4.sh from here, chmod +x, insert USB
    drive, ran it, selected ISO, tells me it can't find my removable
    drive but everything else sees it.
  Output of lsblk:
    sde 8:64 1 7.5G 0 disk 
    ├─sde1 8:65 1 712M 0 part 
    └─sde2 8:66 1 17.2M 0 part 
  —
    Reply to this email directly or view
      it on GitHub.
```

**Comment by thefanclub** [comment]
Issue resolved in version 2.2
